README.md

```diff
@@ -65,9 +66,9 @@ String {
 	replaceAll(search: string, replace: string): string; // Replace all occurrences of search with replace
 	similarity(compare: string): number; // Returns a value between 0 (different) and 1 (same) indicating how similar the string is to compare
 	join(iterate: string[]): string; // Returns the array values seperated by the given divider as a string
-	partition(): string[] // Returns split array, but includes separators
-	toNumber(): number // converts string to number, if not a number returns NaN
-	toBigInt(): number // converts string to BigInt, if not a number returns NaN
+	partiton(): string[]; // Returns split array, but includes separators
```

Comment:
typo should be ``partition``

Reply:
fixed in https://github.com/xNaCly/missing-native-JS-functions/commit/9d2f52e2576c1a6aa6d824cd42bb8925dacfa4c4